### DISCUSSION TITLE
New to Github Organizations- few basic questions

### DISCUSSION THREAD

**Discussion by talc-elad** [community/community#30922] [New to GitHub]
Hi!
I have a new team that is pretty new to Github.
Today, all 5 team members have a free account. We have several current repor which are managed by a team member.
When a new employee joins the team we have to share with him all the current repos. it is time-consuming and not very elegant.

As I understand the organization should solve it.
I need to create an organization. The organization will function as a group of repos and each time a developer joins the team I will add him to the organization. So far I understand it correctly.

**3 questions/issues:**
1. We have the current repos which stored production code. Applications and other business partners are using those repos through their web addresses. They mostly modify JSON files in those repos. Will the repos web address will be changed when I add them to an organization?
2. I am afraid that my business partners will not be able to use the repos inside the organization. Should I add them to the organization? my business partners are mosly editing JSON files in the current repos.
3. The current repos are currently private repos. I created an organization and did not see an option to set the organization as a private organization. If I understand correctly if a private repo is inside an organization, a user without access to the organization will not be able to access the repo? even not read-only access.

Thank you!

**Comment by queenofcorgis** [comment]
Nice to hear you're getting started with GitHub and organizations 😄 

1. The wed address will redirect to the new location. [This doc](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) gives more detail. 
2. Can you elaborate more here? Is there a privacy concern? You have the ability to set [roles](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization) and [teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) in your organization as well as [repository roles](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) that may help with this. 
3. If they are not a part of the organization and the repository is private, the user will not have access to the repo unless you designate them access via roles or outside collaborators. 

Hope this helps - let us know if you have any other questions!

**Reply by talc-elad** [reply]
Thank you for the answer :)
**Regarding #2.**
Today, my business partners edit JSON files inside the current private repos which I am share with them.
For the existing repos:
Let's say I move  a current private repos to the organization. In order to allow my business partner modify those JSONs files I will just need to add them to the organization I created. Am I right?

For the future repos:
For the future repose which will be created inside those organization (and not moved to the organization) in this case 
 In order to allow my business partner modify those JSONs files I will need to add them to the organization I created.


**Regarding 3:**
I pretty confused, what determine the access to the repo inside an organization. I was able to think about 3 cases:
- a private which was moved to an organization- Only user with access to the organization will have access to the repo?
- a private repo which was created inside an organization from Day1- Only user with access to the organization will have access to the repo?
- a **public** repo which was created inside an organization from Day1- Only user with access to the organization will have access to the repo?

Appreciate the help, Tal
